src/mmw/js/src/core/templates/header.ejs

```diff
@@ -13,7 +13,7 @@
                     </a>
                     <ul class="dropdown-menu">
                         <li><a href="#">Profile</a></li>
-                        <li><a href="accounts/logout">Logout</a></li>
+                        <li><a href="/user/logout" class="user-logout" onclick="return false;">Logout</a></li>
```

Comment:
I don't think this really bothers anyone else, but I prefer keep the html pure and then call ```preventDefault``` in the handler.